src/pages/machines/sensor.tsx

```diff
@@ -1,32 +1,16 @@
 /* eslint-disable react/jsx-key */
 import MachineCard from '@/components/reusable/card/machineCard';
-import sensor1 from '../../assets/images/sensor1.png';
-import sensor2 from '../../assets/images/sensor2.png';
-import sensor3 from '../../assets/images/sensor3.png';
-import sensor4 from '../../assets/images/sensor4.png';
-const sensors = [
-  { title: 'Hydraulic Oil temprature', image: sensor1 },
-  { title: 'Hydraulic pump pressure', image: sensor2 },
-  { title: 'Hydraulic oil level', image: sensor3 },
-  { title: 'Spindle oil level', image: sensor4 },
-  { title: 'Spindle oil outlet temprature', image: sensor1 },
-  { title: 'Spindle oil pump pressure', image: sensor2 },
-  { title: 'Coolant oil pump pressure', image: sensor3 },
-  { title: 'Oil level in lube tank', image: sensor4 },
-  { title: 'Lube oil pump pressure', image: sensor1 },
-  { title: 'Hydraulic oil pump pressure', image: sensor2 },
-  { title: 'Hydraulic Oil temprature', image: sensor3 },
-  { title: 'Hydraulic pump pressure', image: sensor4 },
-  { title: 'Hydraulic oil level', image: sensor1 },
-];
+import { ALL_SENSOR_DATA } from '@/utils/machinedata';
+import { useNavigate } from 'react-router-dom';
 
 const SensorPage = () => {
+  const navigate = useNavigate();
   return (
     <div className="flex gap-14 flex-wrap">
-      {sensors.map((sensorData) => (
+      {ALL_SENSOR_DATA.map((sensorData) => (
         <MachineCard
           key={sensorData.title}
-          handleView={() => ''}
+          handleView={() => navigate(sensorData.link)}
```

Comment:
ensure this changes the route

Reply:
yes bro. they have not given any specific screen to view specific sensor. In figma comments they say it should just show the graph page by clicking on it. so as of now i had made it like it navigates to the any sensor graph. if we click "view button".  @chandran-kannan